src/LibraryViewExtension/LibraryViewController.cs

```diff
@@ -172,13 +176,13 @@ private void Browser_LoadError(object sender, LoadErrorEventArgs e)
         {
             System.Diagnostics.Trace.WriteLine("*****Chromium Browser Messages******");
             System.Diagnostics.Trace.Write(e.ErrorText);
+            this.dynamoViewModel.Model.Logger.LogError(e.ErrorText);
```

Comment:
@QilongTang @ramramps this should help us log any failures to the console when the browser tries to start, should be useful.